content/en/docs/reference/glossary/compute-resource.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Resource (compute)
```

Comment:
```suggestion
title: Resource (infrastructure)
```

Writing _compute_ might make some people look for an entry about **Resource (memory)**.